Ginger/GingerCoreNET/UserCofig/UserProfile.cs

```diff
@@ -449,6 +452,7 @@
                 Reporter.ToLog(eLogLevel.ERROR, "Error occurred while saving Recent App-Agents Mapping for User Profile save", ex);
             }
             RepositorySerializer.SaveToFile(this, UserProfileFilePath);
+            SavePasswords();
```

Comment:
Need to check if this is called when running in CLI mode. If yes then it can cause issue on linux